src/AzureAuth/Commands/Ado/CommandPat.cs

```diff
@@ -25,6 +25,7 @@ namespace Microsoft.Authentication.AzureAuth.Commands.Ado
     /// An ADO Command for creating or fetching, and returning Azure Devops PATs.
     /// </summary>
     [Command("pat", Description = "Create and cache Azure Devops Personal Access Tokens (PATs) using encrypted local storage.")]
+    [Subcommand(typeof(Pat.CommandScopes))]

```

Comment:
I'm very unfamiliar with CLI standards, so genuinely just curious. Is there a reason for picking a subcommand as opposed to creating a new argument, or having this live under -h?

Reply:
Great question, I intentionally did not add this as an option to the existing pat command, because that would create a divergence in it's logic, complicating it's option validation and the main execution code all together. It also would violate single responsibility. 

I didn't put this in the help, because generally you want your help text to be as succinct as possible. It's not full-on documentation and should be quick to read. The output list of scopes is quite long and would obfuscate the rest of the help, which for this command, does have several required options.

Reply:
That makes perfect sense! Thanks for the explanation 😊